cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -441,16 +836,287 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Unwind",
       text =
-        """Takes a list of values and returns one row per item in the list.""".stripMargin,
-      queryText = """UNWIND range(1, 5) as value return value;""",
+        """The `Unwind` operator returns one row per item in a list.""".stripMargin,
+      queryText = """UNWIND range(1, 5) as value return value""",
       assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("Unwind"))
     )
   }
 
+  @Test def apply() {
+    profileQuery(
+      title = "Apply",
+      text =
+        """`Apply` works by performing a nested loop.
```

Comment:
All the different `Apply` operators share the same basic functionality - they perform a nested loop, taking a single row from the LHS, and through the `Argument` operator on the RHS, executes the RHS subtree. What happens with these results is what the variation between the different `Apply` versions. 

The unadorned `Apply` operator takes the row produced from the RHS, which by now contains data from both sides, and yields it.

Reply:
👍